dv/uvm/core_ibex/riscv_dv_extension/csr_description.yaml

```diff
@@ -142,47 +142,52 @@
 # fields need to be constrained such that their value after every operation is within the allowed
 # range of values - too much complexity for the current format
 # MSTATUS
-#- csr: mstatus
-#  description: >
-#    Controls hart's current operating state
-#  address: 0x300
-#  privilege_mode: M
-#  rv32:
-#    - field_name: tw
-#      description: >
-#        Timeout Wait (WFI from U-mode will trap to M-mode)
-#      type: WARL
-#      reset_val: 0
-#      msb: 21
-#      lsb: 21
-#    - field_name: mprv
-#      description: >
-#        Modify Privilege (Loads and stores use MPP for privilege checking)
-#      type: WARL
-#      reset_val: 0
-#      msb: 17
-#      lsb: 17
-#    - field_name: mpp
-#      desription : >
-#        Previous privilege mode
-#      type: R
-#      reset_val: 0
-#      msb: 12
-#      lsb: 11
-#    - field_name: mpie
-#      description: >
-#        Previous value of interrupt-enable bit
-#      type: WARL
-#      reset_val: 1
-#      msb: 7
-#      lsb: 7
-#    - field_name: mie
-#      description: >
-#        M-mode interrupt enable
-#      type: WARL
-#      reset_val: 0
-#      msb: 3
-#      lsb: 3
+- csr: mstatus
+  description: >
+    Controls hart's current operating state
+  address: 0x300
+  privilege_mode: M
+  rv32:
+    - field_name: tw
+      description: >
+        Timeout Wait (WFI from U-mode will trap to M-mode)
+      type: WARL
+      reset_val: 0
+      msb: 21
+      lsb: 21
+    - field_name: mprv
+      description: >
+        Modify Privilege (Loads and stores use MPP for privilege checking)
+      type: WARL
+      reset_val: 0
+      msb: 17
+      lsb: 17
+    - field_name: mpp
+      desription : >
+        Previous privilege mode
+      type: WARL
+      reset_val: 0
+      msb: 12
+      lsb: 11
+      warl_legalize: |
+        if val_in == 0b11 or val_in == 0b00:
```

Comment:
This seems correct to me, since MPP is WARL and based on the following line in the spec:
> If the machine provides only U and M modes, then only a single hardware storage bit is required to represent either 00 or 11 in MPP.